examples/fishjam-chat/index.js

```diff
@@ -1,17 +1,5 @@
-/**
- * @format
- */
+import registerRootComponent from 'expo/build/launch/registerRootComponent';
 
-import { AppRegistry } from 'react-native';
 import App from './App';
-import { name as appName } from './app.json';
-import { LogBox } from 'react-native';
-import notifee from '@notifee/react-native';
 
-LogBox.ignoreLogs(['new NativeEventEmitter']);
-
-notifee.registerForegroundService((notification) => {
-  return new Promise(() => {});
-});
-
-AppRegistry.registerComponent(appName, () => App);
+registerRootComponent(App);
```

Comment:
**question:** Why do we still need index.js?

Reply:
Because in monorepo environment, expo still has some issues with finding right file to start app. This was easier workaround that worked for me 🤷